frontend/src/components/activity/dialog/FormScheduleEntryItem.vue

```diff
@@ -283,4 +292,67 @@ export default {
   padding-top: 12px;
   margin-top: 4px;
 }
+
+.area-startdate {
+  grid-area: startdate;
+}
+
+.area-starttime {
+  grid-area: starttime;
+}
+
+.area-enddate {
+  grid-area: enddate;
+}
+
+.area-endtime {
+  grid-area: endtime;
+}
+
+.area-dash {
+  display: none;
+  grid-area: dash;
+}
+
+@media (min-width: 768px) {
+  .area-dash {
+    display: block;
+  }
+  .area-duration {
+    width: 130px;
+    margin-left: 0;
+  }
+}
+
+.area-duration {
+  grid-area: duration;
+  margin-left: 8px;
+}
+
+.hide-control:not(:hover):not(:focus):not(:focus-within):not(:has(.error--text))
+  :deep(.v-input__control) {
+  display: none;
+}
+
+.area-delete {
+  grid-area: delete;
+}
+
+.ec-form-schedule-entry-item {
+  display: grid !important;
+  row-gap: 4px;
+  column-gap: 0px;
+  grid-template-areas:
+    'startdate starttime duration'
+    'enddate endtime delete';
+  grid-template-columns: auto auto 1fr;
+}
+
+@media (min-width: 768px) {
```

Comment:
did you repeat @media (min-width: 768px) { on purpose?

Reply:
i thought it makes more sense to keep the different media breakpoints of a class close to the mobile definition